docs/Dirb.md

```diff
@@ -0,0 +1,36 @@
+# How to install and use Dirb for Mac OS
+
```

Comment:
Before starting a new subtopic, it would be nice to have a paragraph describing the main objective of this document. What do you think of this: 

`The main goal of this document is to describe how Dirb can be installed, configured and used on a OSX environment. This tool must only be used in SecDevLabs apps and on systems you have proper authorization.`

Reply:
It looks great! I wrote something similar to this further below, but I think it looks best on top!

Reply:
My bad! I think there is a typo here: `be used in SecDevLabs apps and on systems` -> `be used on SecDevLabs apps and on systems`